BuffBot/dbUnitTest.py

```diff
@@ -6,23 +6,38 @@
 class DatabaseTest(unittest.TestCase):
     # Prepare the relevant table for testing by deleting all rows in it.
     def emptyTable(self, table):
-        self.conn = sqlite3.connect('test123.db')
+        self.conn = sqlite3.connect("test123.db")
         sql = 'delete from {} '.format(table)
         self.conn.execute(sql)
         self.conn.commit()
         self.conn.close()
 
     def setUp(self):
-        self.emptyTable("game_restriction")
-        # Create a Database object we can call functions on.
+        self.emptyTable('game_restriction')
+        self.emptyTable('members')
+        # Create a Database object to call the relevant functions.
         self.db = Database()
         # Insert dummy data into the database.
         self.db.flag_gaming_channel("1", "Hearthstone", 1)
+        self.db.insert_coins(85431603408420864, 4000, 'Lenny#1112')
+        self.db.insert_coins(235892294857916417, 23, 'Christian#1111')
+        self.db.insert_coins(278286021903515656, 20000, 'BuffBot#0334')  # Mocking the bot user.
```

Comment:
Yes, that is possible. However, if we would like to wipe all the changes made by the test to the database, including the emtpytable(table), we would have to  querry all the actual data in the relevant testing tables: 
![image](https://cloud.githubusercontent.com/assets/16694949/25685331/0cff5712-3067-11e7-9aca-25bcf507f84a.png)


, save this result to a variable in setUp(). Then we would have to querry the content of this variable back into the DB in a tearDownClass function. 
In cases where there is a lot of data in these tables, I imagine this could eat a huge amount of RAM, _but do we care?_. 

There are of course other options, which may be much better than this concept I provided. If so, do tell! 
Could it be an idea to simply have an own branch designated for testing?